packages/starknet-snap/src/utils/keyPair.ts

```diff
@@ -1,7 +1,7 @@
 import type { BIP44AddressKeyDeriver } from '@metamask/key-tree';
 import { getBIP44AddressKeyDeriver } from '@metamask/key-tree';
 import { utils } from 'ethers';
-import { number, ec } from 'starknet_v4.22.0';
+import { num as numUtils, ec } from 'starknet';
```

Comment:
this is very risky change, are we confirm the address is always the same as before

Reply:
Ok, because of potential big implication we rollback this and will be done later on an other PR